cmd/main.go

```diff
@@ -50,8 +52,10 @@ var (
 	bqDataset   = flag.String("bq_dataset", "", "DataCommons BigQuery dataset.")
 	schemaPath  = flag.String("schema_path", "", "The directory that contains the schema mapping files")
 	// Base Bigtable Cache
-	useBaseBt     = flag.Bool("use_base_bt", true, "Use base bigtable cache")
-	baseTableName = flag.String("base_table", "", "Base cache Bigtable table.")
+	useBaseBt         = flag.Bool("use_base_bt", true, "Use base bigtable cache")
+	baseTableName     = flag.String("base_table", "", "Base cache Bigtable table.")
+	useImportGroup    = flag.Bool("use_import_group", false, "Use multiple base tables from import group")
+	importGroupTables = flag.String("import_group_tables", "", "A JSON string of import group tables")
```

Comment:
... in place of baseTableName. This is relevant only when --use_import_group is true... ?

Reply:
Instead of "A JSON string...", would it be easier just to make it a comma-separated list of strings?

Reply:
Done. And the json content is read from bigtable_import_groups.json, not manually input.. So it's easier to load from that. That way we can add more configs to the json as well.